src/bin/setup.rs

```diff
@@ -0,0 +1,36 @@
+use schedule_bot::data::{unpack, Schedule, Subject};
+use sqlx::SqlitePool as Pool;
+
+#[tokio::main]
+async fn main() {
+    pretty_env_logger::init();
+
+    // could've used Clap, but I literally need just 1 argument
+    let mut args = std::env::args();
+    args.next(); // skip the first element
+    let url = args.next().expect("Missing database url");
```

Comment:
Why not to use `std::env::var()` ?
I think it both cleaner and proper to use, since user may have the first environment variable supposed to be used by another program

Reply:
`std::env::args()` is not `std::env::vars().next()`, it is a first cmd argument. If I understood your question correctly.

Reply:
> `std::env::args()` is not `std::env::vars().next()`, it is a first cmd argument. If I understood your question correctly.

Oh, wait, I thought he was reading an environment variable 😅😅😅

Now it makes sense if it's a command argument. Sorry for this misconfusion 😅

Reply:
No worries!